packages/client/src/App.js

```diff
@@ -12,6 +15,9 @@ function App() {
       <Router>
         <Routes>
           <Route path="/" element={<LandingPage />} />
+          <Route exact path="/about-us" element={<AboutUs />} />
+          <Route exact path="/contact-us" element={<ContactUs />} />
+          <Route exact path="/login" element={<Login />} />
           <Route path="*" element={<PageNotFound />} />
           <Route path="/products/:id" element={<ProductView />} />
           <Route path="/collections" element={<CollectionsPage />} />
```

Comment:
I think, at the Navigation bar, this link to="/our-collections", would be great if you be able to change to link to=" /collections" at the navigation component.

Reply:
How about /signup?

Reply:
@Hailegebriel22  good insight 👍🏽 
We discussed it with @Hanouj , I asked to stick to what the task asks for and "merge" login and signup into one for now. 
<img width="928" alt="image" src="https://user-images.githubusercontent.com/64137994/204385945-444e4801-6ec5-4695-9edd-d5f17cac266d.png">